src/backend/distributed/citus--5.1-5--5.1-6.sql

```diff
@@ -0,0 +1,67 @@
+CREATE FUNCTION pg_catalog.master_drop_sequences(sequence_names text[],
+												 node_name text,
+												 node_port bigint)
+    RETURNS bool
+    LANGUAGE C STRICT
+    AS 'MODULE_PATHNAME', $$master_drop_sequences$$;
+COMMENT ON FUNCTION pg_catalog.master_drop_sequences(text[], text, bigint)
+    IS 'drop specified sequences from a node';
+
+REVOKE ALL ON FUNCTION pg_catalog.master_drop_sequences(text[], text, bigint) FROM PUBLIC;
+
+CREATE OR REPLACE FUNCTION pg_catalog.citus_drop_trigger()
+    RETURNS event_trigger
+    LANGUAGE plpgsql
+    SET search_path = pg_catalog
+    /* declared as SECURITY DEFINER in upgrade script */
```

Comment:
Uh, are you sure this doesn't reset that property, given that's an earlier upgrade script? I'd much rather copy that here.